tests/ts/kill/decode

```diff
@@ -53,14 +53,19 @@ ACK=
 	    # Sending one more USR1 is for making the signal pending state.
 	    "$TS_CMD_KILL" -USR1 "$PID"
 	    "$TS_CMD_KILL" -d "$PID" | {
-		if [[ $("$TS_CMD_KILL" --list=34) == RT0 ]]; then
+		SIGRTMIN=$($TS_HELPER_SYSINFO SIGRTMIN)
+		if [[ $("$TS_CMD_KILL" --list=$SIGRTMIN) == RT0 ]]; then
```

Comment:
Looking at this again, we already have `kill --list=RTMIN+0`. Would this be sufficient?
Sorry for making you do extra work.